guide/ops/locations/_more-clouds.md

```diff
@@ -26,404 +26,3 @@ is included below. You may also find these sources helpful:
   sometimes required for various clouds.
 
 
```

Comment:
It feels with all this content removed this file isn't really doing anything any more and could be removed